tools/compiletest/src/runtest.rs

```diff
@@ -272,7 +272,9 @@ impl<'test> TestCx<'test> {
             .arg("kani")
             .arg("--target-dir")
             .arg(self.output_base_dir().join("target"))
-            .current_dir(parent_dir);
+            .current_dir(parent_dir)
+            .arg("--coverage")
```

Comment:
Isn't this turning on coverage for all `cargo kani` tests? Is this intentional?

Reply:
This was intended as a temporary temporary